config/initializers/gem-plugin_config/sanitizer_config.rb

```diff
@@ -35,7 +35,9 @@ module Config
         'blockquote' => {'cite' => ['http', 'https', :relative]},
         'img' => {'src' => ['http', 'https', :relative]},
         'q' => {'cite' => ['http', 'https', :relative]}
-      }
+      },
+
```

Comment:
Could we have a `TODO` here reminding us to remove this when we actually update the gem (the issue for that is AO3-5801)?